wdlc.sh.in

```diff
@@ -425,6 +471,21 @@ while [ "${#}" -gt 0 ]; do
             shift 2
             ;;
 
+        -MF)
+            dependency_file_path=${2}
+            shift 2
+            ;;
+
+        -MT)
+            dependency_file_target=${2}
+            shift 2
+            ;;
+
+        -MP)
+            emit_phony_target=true
```

Comment:
Style: We tend to use 0/1 rather than true/false elsewhere.

Reply:
There was another example of boolean usage in the file, so I just copy/pasted it. This will likely have to be a bigger conversation around style/consistency :)